plover/registry.py

```diff
@@ -54,6 +55,17 @@ def update(self):
             entrypoint_type = 'plover.%s' % plugin_type
             for entrypoint in pkg_resources.iter_entry_points(entrypoint_type):
                 self.register_plugin(plugin_type, entrypoint)
+            if sys.platform.startswith('darwin'):
```

Comment:
Yeah, I've been thinking about that. I've wanted to replace our use of OS X throughout the code base.

macOS is kind of an obnoxious name, though. Especially the idea of `macoskeyboardcontrol`…but "mac" alone seems fine to me. Same character count as OS X and not going to be outdated.